src/Bridges/Nette/Bridge.php

```diff
@@ -52,7 +52,7 @@ public static function renderLatteError(?\Throwable $e): ?array
 			$lines = file($file);
 			if (preg_match('#// source: (\S+\.latte)#', $lines[1], $m) && @is_file($m[1])) { // @ - may trigger error
 				$templateFile = $m[1];
-				$templateLine = preg_match('#/\* line (\d+) \*/#', $lines[$e->getLine() - 1], $m) ? (int) $m[1] : null;
+				$templateLine = preg_match('#/\* line (\d+) \*/#', $lines[$e->getLine() - 1] ?? '', $m) ? (int) $m[1] : null;
```

Comment:
Imho this is more readable `$e->getLine() && preg_match('#/\* line (\d+) \*/#', $lines[$e->getLine() - 1], $m) ? (int) $m[1] : null;`

Reply:
Fixed, thanks.